js/dlg/message-box.js

```diff
@@ -102,6 +106,28 @@ messageBox.show = async ({
       scroll() {
         scrollTo(messageBox._blockScroll.x, messageBox._blockScroll.y);
       },
+      mouseDown(event) {
+        clickX = event.clientX;
+        clickY = event.clientY;
+
+        const element = $('#message-box > div');
+        positionX = element.offsetLeft;
+        positionY = element.offsetTop;
+        dragging = true;
+      },
+      mouseUp() {
+        dragging = false;
+      },
+      mouseMove(event) {
+        if (dragging) {
+          const x = positionX + event.clientX - clickX;
+          const y = positionY + event.clientY - clickY;
+
+          const element = $('#message-box > div');
+          element.style.left = x + 'px';
```

Comment:
AFAIK It's better to use CSS transform `translate` for shifting without changing layout.